capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java

```diff
@@ -93,7 +93,7 @@ public static Entity getUser() {
         }
     }
     return null;
-  }
+  }}
```

Comment:
is this just a stray curly brace or does it just allow it to run?

Reply:
That might have been an accident. I'll get rid of it